README.md

```diff
@@ -151,6 +151,9 @@ Run `"make"` on macOS, linux, Windows (or in fact RPi also) host to build these
   line
 - `target/aarch64-unknown-linux-gnu/release/piglet` - Headless version for Pi with GPIO, can be run natively from RPi
```

Comment:
Add something about the architecture, and the Pi models that use it - to help people know which arch they need for their Pi?